tests/test_rkllama_default_port.py

```
@@ -0,0 +1,77 @@
+# ---------------------------------------------------------------------------
+# (c) Shell script regression guard
+# ---------------------------------------------------------------------------
+
+class TestInstallScriptDefault:
+    def test_shell_default_is_7833(self):
+        script = pathlib.Path(__file__).parents[1] / "scripts" / "install-rknpu.sh"
+        assert script.exists(), "scripts/install-rknpu.sh not found"
+        content = script.read_text()
+        assert "TAOS_RKLLAMA_PORT:-7833" in content, (
+            "install-rknpu.sh default port is not 7833; "
+            "found: " + repr([l for l in content.splitlines() if "TAOS_RKLLAMA_PORT" in l])
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)